fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1156,8 +1155,7 @@ void writeMutation(CommitBatchContext* self, int64_t tenantId, const MutationRef
 		self->toCommit.writeTypedMessage(mutation);
 	} else {
 		Arena arena;
```

Comment:
nit: for safety add below assert
` assert(server_knobs->enable_encryption)`

Reply:
line 1150 above makes sure that.